css-two-column-layout/styles.css

```diff
@@ -0,0 +1,145 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-half {
+  width: 50%;
+}
+
+.smart-software-section {
+  display: flex;
```

Comment:
I see you are using `display: flex` for each of these sections. When you are using layout classes to control your app layout, `display: flex` is usually an indicator that you need another `row`. You should put a `row` class element inside the `column` these classes are being applied to, then have additional `column` classes for the content inside that `row` It's always okay to have additional `row`s inside of a `column` class element 🤓